demisto_sdk/commands/common/tools.py

```diff
@@ -822,15 +822,15 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
         if 'brandName' in _dict and 'transformer' in _dict:
             return FileType.OLD_CLASSIFIER
 
-        if 'transformer' in _dict and 'keyTypeMap' in _dict:
-            return FileType.CLASSIFIER
+        if ('transformer' in _dict and 'keyTypeMap' in _dict) or 'mapping' in _dict:
+            if _dict.get('type') == 'classification':
```

Comment:
```suggestion
            if _dict.get('type') and _dict.get('type') == 'classification':
```

let's be defensive here

Reply:
not really necessary as `None != 'classification'`